app/src/main/java/openfoodfacts/github/scrachx/openfood/views/HistoryScanActivity.java

```diff
@@ -131,7 +131,7 @@ public void exportCSV() {
             } else {
                 writer = new CSVWriter(new FileWriter(filePath));
             }
-            String[] headers = {"Barcode", "Name", "Brands"};
+           String[]  headers = res.getStringArray(R.array.headers);
```

Comment:
The build fails here: https://travis-ci.org/openfoodfacts/openfoodfacts-androidapp/builds/343922899#L2985

You should replace `res` by `getResources()` to make it work